### ISSUE TITLE
Bots for Team Training

### ISSUE THREAD

**Issue by Kozlov98** [haxball/haxball-issues#58]
Often the teams (or individual players too) did not find a free host to play or train, so it would be a great help to combine Bots to train alone or with your team. I know it is difficult to program, but it would be good if at least there is for the 3v3 or 4v4 an archer and forwards / defenders.

Thx for read - Kozlov

**Comment by matixxx0** [comment]
Maps traning done

**Comment by Leo-MapCreator** [comment]
Hi basro, i dont know if you already played HaxArena, they have this feature, you can choose bots to play for red or blue, i dont know if it would be great to create a partnership with them, you would be able to enhace HaxB a lot faster, it's just and idea, it have to be viable for you though

**Comment by helppe** [comment]
thanks for the news leo.
i didn't know about haxarena :o

**Comment by Blaze52** [comment]
Yes, having bots in this game would be absolutely awesome. And HaxArena would lose the one thing that sets them apart from basro's haxball, so that excuse of a game can die.
